docs/index.md

```diff
@@ -55,4 +55,4 @@ Terms of Service.
 - **credentials** (String) Either the path to or the contents of a service account key file in JSON format you can manage key files using the Cloud Console).  If not provided, the application default credentials will be used.
 - **customer_id** (String) The customer id provided with your Google Workspace subscription. It is found in the admin console under Account Settings.
 - **impersonated_user_email** (String) The impersonated user's email with access to the Admin APIs can access the Admin SDK Directory API.
-- **oauth_scopes** (List of String) The list of the scopes required for your application (for a list of possible scopes, see [Authorize requests](https://developers.google.com/admin-sdk/directory/v1/guides/authorizing))
+- **oauth_scopes** (List of String) The list of the scopes required for your application (for a list of possible scopes, see [Authorize requests](https://developers.google.com/admin-sdk/directory/v1/guides/authorizing)). This can be set using the `GOOGLEWORKSPACE_OAUTH_SCOPES` environment variable by passing the scopes as a list of strings, for example: `GOOGLEWORKSPACE_OAUTH_SCOPES="https://www.googleapis.com/auth/admin.directory.group,https://www.googleapis.com/auth/admin.directory.user"`.
```

Comment:
This is generated by the `Description` in `provider.go`, so you'll want to update it there and then run `make generate`